Scripts/script-QRadarFullSearch.yml

```diff
@@ -0,0 +1,93 @@
+commonfields:
+  id: QRadarFullSearch
+  version: -1
+name: QRadarFullSearch
+script: |-
+  res = [];
+
+  var search_done = false;
+  var error = false;
+
+  var timeout = ( 'timeout' in args) ? parseInt(args.timeout) : 600;
+  var interval = ( 'interval' in args) ? parseInt(args.interval) : 10;
+
+  var search_args = {};
+  if ('fields' in args){
+      search_args.fields = args.fields;
+  }
+  if ('range' in args){
+      search_args.range = args.range;
+  }
+
+  if ('headers' in args){
+      search_args.headers = args.headers;
+  }
+
+  //submit query, retrive  search_id
+  var query_res = executeCommand("qradar-searches", args);
+  if (isError(query_res)){
+      return query_Res;
+  }
+  else {
+      search_id = dq(query_res[0], "Contents.search_id");
+      search_args.search_id = search_id;
+      //polling stage
+
+      var sec = 0;
+      while ((sec < timeout) && !error){
+          status_res = executeCommand("qradar-get-search", search_args);
+          if (isError(status_res[0])){
+              return status_res;
+          }
+
+          var q_status = dq(status_res[0], "Contents.status");
+
+          if (q_status && q_status in ['WAIT', 'EXECUTE', 'SORTING']){
+              log('search id is {0} search status is {1}'.format(search_id,q_status));
+          }
+          else if (q_status && q_status == 'COMPLETED'){
+              search_done = true;
+          }
+          else{
+              error = true;
+              return {"Type" : entryTypes.error, "ContentsFormat" : formats.text, "Contents" : 'An Error occurred during the search process. search status={0}.'.format(status)};
```

Comment:
No need for "Type" in JS, Type: is enough (sets it as a string key automatically)